### ISSUE TITLE
Crash macOS on certain operations

### ISSUE THREAD

**Issue by zcysxy** [withfig/fig#1431]
### Sanity checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have typed 'git ' and included a screenshot of the Debugger (click the Fig icon in the macOS menu bar)

### Description:

I'm on macOS Monterey 12.3.1. When Fig is toggled on in Privacy > Accessibility, some operations in **other apps** will crash the computer. These operations include:

* Open PDF in VS Code
* Open PDF in Edge
* Close figure window in MATLAB

When the accessibility is turned off, all these operations will not cause crashes.

This problem has bothered me for a long time since I had no clue it has something to do with Fig. Until one day I saw [this post](https://www.mathworks.com/matlabcentral/answers/646743-matlab-2020b-crashes-everytime-i-close-a-figure-manually). Then I tested these operations with Fig accessibility on and off, and found the issue.

---

Debugger:

<img width="421" alt="image" src="https://user-images.githubusercontent.com/48207708/167613710-07e470ee-2a87-470b-94e3-f9a499283e4e.png">

### Environment

```bash
# Fig Diagnostics
## Fig details:
  - Fig version: Version 1.0.57 (B440) [ABC]
  - Bundle path: /Applications/Fig.app
  - Autocomplete: true
  - Settings.json: true
  - Accessibility: true
  - Number of specs: 0
  - Symlinked dotfiles: false
  - Only insert on tab: false
  - Keybindings path: 
  - Installation Script: true
  - PseudoTerminal Path: /Users/ce/.local/bin:/Applications/MATLAB_R2021b.app/bin:/Applications/MATLAB_R2021b.app/bin/maci64:/Users/ce/.nvm/versions/node/v17.5.0/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Users/ce/.fig/bin:/Users/ce/.local/bin:/opt/homebrew/opt/fzf/bin
  - SecureKeyboardInput: false
  - SecureKeyboardProcess: <none>
## Hardware Info:
  - Model Name: MacBook Pro
  - Model Identifier: MacBookPro18,3
  - Chip: Apple M1 Pro
  - Cores: 8 (6 performance and 2 efficiency)
  - Memory: 16 GB
## OS Info:
  - macOS 12.3.1 (21E258)
## Environment:
  - User Shell: /bin/zsh
  - Current Directory: /Users/ce
  - CLI Installed: true
  - Executable Location: /Users/ce/.local/bin/fig
  - Current Window ID: 12116/% (com.googlecode.iterm2)
  - Active Process: zsh (93650) - not a tty
  - Installed via Brew: true
  - Environment Variables:
    - TERM_SESSION_ID=w0t0p0:F06CD06B-4DCB-41CC-9C20-DCA565ED04ED
    - PATH=/Users/ce/.local/bin:/Applications/MATLAB_R2021b.app/bin:/Applications/MATLAB_R2021b.app/bin/maci64:/Users/ce/.nvm/versions/node/v17.5.0/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Users/ce/.fig/bin:/Users/ce/.local/bin:/opt/homebrew/opt/fzf/bin
    - TERM=xterm-256color
    - FIG_INTEGRATION_VERSION=8
    - FIG_TERM=1
    - FIG_TERM_VERSION=4.3.0
    - FIG_PID=93650
## Integrations:
  - SSH: false
  - TMUX: false
  - iTerm: installed!
  - Hyper: application is not present.
  - Visual Studio Code: installed!
  - Docker: false
```

**Comment by mschrage** [comment]
Unfortunately, this is a bug with Matlab. Nothing we can do to resolve it on our end.

**Comment by SeparateRecords** [comment]
@zcysxy how did opening a PDF in VSCode or Edge cause a crash? Can you describe how to reproduce this?

**Comment by zcysxy** [comment]
@mschrage it's not a bug with MATLAB, it happens in other scenarios. I just tried to provide as much info as I can, so I posted the MATLAB forum link.

Hi @SeparateRecords I recorded a video of the issue on VS Code when previewing LaTeX pdfs. The issue also happens when previewing Markdown pdfs using Markdown Preview Enhanced.

![vscode](https://user-images.githubusercontent.com/48207708/167770247-55451c19-669c-4f3c-8a20-affad6252586.gif)

The issue on Edge happens when you open multiple pdfs from the Google search result, and switch to them before they are fully loaded. It's hard to catch the issue on Edge cause it happens randomly.
